templates/vue/src/components/tyde/activities/CircleOfSupport/CommunityView/onboarding/ObFinishView.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <cos-modal show>
+    <b-container class="modal-cont">
```

Comment:
Please avoid shortening names where possible. `modal-container` would be a perfectly fine and more clear class name here.